### PR TITLE
update release data

### PR DESCRIPTION
**Pull request by VincyZhang**
## Type of Change
documentation

## Description

update release data

### CODE REVIEW

File: docs/release_data.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+
+2. [LLM Runtime Inference based on Pytorch](#llm-runtime-inference-based-on-pytorch)
+
+    2.1 [LLMs](#llms)
+
+    2.2 [Stable Diffusion](#stable-diffusion)
+
+    2.3 [Electra](#electra)
+
+3. [LLM Runtime Inference based on ITREX.cpp](#llm-runtime-inference-based-on-itrexcpp)
```
Comment (kevinintel) [inline docs/release_data.md:13]: LLM Runtime inference

File: docs/release_data.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
+Validated Model Performance
+============
+1. [LLM Quantization](#llm-quantization)
+
+2. [LLM Runtime Inference based on Pytorch](#llm-runtime-inference-based-on-pytorch)
```
Comment (kevinintel) [inline docs/release_data.md:5]: based on python mode

File: docs/release_data.md (hunk 1 above)
Comment (hshen14) [reply docs/release_data.md:13]: No, LLM Runtime (GGML-Compatible)

File: docs/release_data.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+| pytorch   | llama_7b      | NeelNanda/pile-10k | 13.56                    | 58.55%            | 4.4                      | 73.61% | 3.09 | \-20.46% |
+| pytorch   | gpt_j_6b      | NeelNanda/pile-10k | 10.76                    | 67.59%            | 4.38                     | 68.31% | 2.46 | \-1.05% |
+| pytorch   | flan_t5_large | samsum             | 69.75                    | 46.25 (rougeLsum) | 33.16                    | 47.67 (rougeLsum) | 2.1 | \-2.99% |
+| pytorch   | gpt_neox_clm  | wikitext           | 1.47                     | 4.04 (eval_loss)  | 0.65                     | 3.52 (eval_loss) | 2.27 | \-14.78% |
+| pytorch   | gpt_j_6b_clm  | wikitext           | 0.86                     | 3 (eval_loss)     | 0.28                     | 2.34 (eval_loss) | 3.1 | \-28.67% |
+| onnx      | whisper_large | lambda-openai      | 2.11                     | 97.07%            | 1.13                     | 96.96% | 1.87 | 0.12% |
+
+
+
+## LLM Runtime Inference based on Pytorch
```
Comment (hshen14) [inline docs/release_data.md:71]: See above

File: docs/release_data.md [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+### LLMs
+
+
+| Framework | Model                 | Input    | Output | Steps    | INT8      | FP32      | BF16      | FP8       | INT8/FP32 | BF16/FP32 | FP8/FP32 |
+| --------- | --------------------- | -------- | ------ | -------- | --------- | --------- | --------- | --------- | --------- | --------- | -------- |
+| pytorch   | gpt-neox-20b          | 32       | 32     |          | 9283 (ms) |           |           |           |           |           |          |
+| pytorch   | dolly-v2-3b           | 32       | 32     |          | 3191 (ms) | 3798 (ms) | 2689 (ms) | 1.19x     | 1.41x     |           |
+| pytorch   | stable_diffusion_v2_1 | 512\*512 | 20     |          | 16.98 (s) | 2.83 (s)  |           | 6.00x     |           |
+| pytorch   | Steps = 20            |          |        |          |           |           |           |           |           |           |
+| pytorch   | stable_diffusion_v1_5 | 512\*512 | 20     | 2.18 (s) | 10.94 (s) | 2.74 (s)  | 5.01x     | 3.99x     |           |
```
Comment (hshen14) [inline docs/release_data.md:85]: stable diffusion is not llm strictly speaking, making the table not unified. 

move to Stable Diffusion section with accuracy table and performance table

**Comment by hshen14** [comment]
please check the MD UI, which does not show correctly 
image

File: docs/release_data.md [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+
+Pytorch: 2.0.1+cpu
+
+Intel Extension for Pytorch: 2.0.100+cpu
+
+Intel Neural Compressor: 2.3
+
+
+|           |               |                    | INT8                     | FP32              | INT8/FP32                |
+| --------- | ------------- | ------------------ | ------------------------ | ----------------- | ------------------------ |
```
Comment (kevinintel) [inline docs/release_data.md:54]: can't view int8 acc, please add  ------------- |

**Review by kevinintel — APPROVED** [review]
LGTM

**Pull request merged**